lucene-jbossmodules-integrationtests/pom.xml

```diff
@@ -86,6 +86,25 @@
             <artifactId>lucene-analyzers-common</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.lucene-jbossmodules</groupId>
+            <artifactId>lucene-jbossmodules</artifactId>
+            <type>pom</type>
+            <scope>provided</scope>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate.lucene-jbossmodules</groupId>
+            <artifactId>lucene-jbossmodules</artifactId>
+            <type>zip</type>
+            <version>${project.version}</version>
+            <exclusions>
```

Comment:
Fair enough, I guess we can do that every time we depend on a feature pack instead of declaring the dependencies as provided. Less convenient, but it'll do.